File: 3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree.py
# Last updated: 9/3/2025, 10:46:08 AM
class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        oper=0
        for i in nums:
            if i%3!=0:
                oper=oper+1
                i=i+1
        return oper

File: 3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree/3476-FindMinimumOperationsToMakeAllElementsDivisibleByThree.java
// Last updated: 9/3/2025, 10:46:12 AM
class Solution {
    public int minimumOperations(int[] nums) {
       int oper=0;
        for(int num:nums){
            int rem=num%3;
            if(rem==1){
                oper+=1;
            }
            else if(rem==2){
                oper+=1;
            }
        }
     
        return oper;
    }
}